kotlin/map-rotation/README.md

```diff
@@ -0,0 +1,28 @@
+# Map rotation
+
+Rotate a map.
+
+![Image of map rotation](map-rotation.png)
+
+## Use case
+
+A user may wish to view the map in an orientation other than north-facing.
+
+## How to use the sample
+
+Use the progress bar to rotate the map. If the map is not pointed north, the progress bar will show the heading relative to north in degrees. Slide the progress bar to zero to set the maps's heading back to north.
```

Comment:
the use of "progress bar" is a bit confusing, we can just say "slider" instead.

Reply:
yes, good point